db/mysql/client.go

```diff
@@ -52,18 +52,28 @@ func NewClient(conf db.Option) (db.DataStore, error) {
 	return &client{db: cli}, nil
 }
 
-func (c client) SaveEmployee(employee *models.Employee) (string, error) {
+func (c client) AddEmployee(employee *models.Employee) (string, error) {
```

Comment:
We didn't need to split this into to function... both add and update apis can use the `SaveEmployee` right?

Reply:
i removed the save employee and created two functions add and update